app/pb_kits/playbook/pb_popover/docs/_popover_list.jsx

```diff
@@ -35,11 +35,11 @@ const PopoverWithButton = () => {
         show={showPopover}
     >
       <List xpadding>
-        <ListItem>{'Popularity'}</ListItem>
-        <ListItem>{'Title'}</ListItem>
-        <ListItem>{'Duration'}</ListItem>
-        <ListItem>{'Date Started'}</ListItem>
-        <ListItem>{'Date Ended'}</ListItem>
+        <ListItem><a><a>{'Popularity'}</a></a></ListItem>
```

Comment:
Double wrapped <a>, also does the console throw errors when the anchors don't have any attributes?

Reply:
This is just a fuck up on my part